packages/daimo-common/src/model.ts

```diff
@@ -78,3 +78,13 @@ export const zTrackedRequest = z.object({
 });
 
 export type TrackedRequest = z.infer<typeof zTrackedRequest>;
+
+export const zKeyData = z.object({
+  key: zHex, // DER Format
+  addedAt: z.number(),
+  removedAt: z.number().optional(),
+  keyIndex: z.number().optional(),
```

Comment:
can be mandatory now

+ `lastUsedAt: z.number().optional()` lgtm

Reply:
done